pandas/io/pytables.py

```diff
@@ -3314,12 +3312,12 @@ def data_orientation(self):
     def queryables(self) -> Dict[str, Any]:
         """ return a dict of the kinds allowable columns for this object """
 
+        # mypy doesnt recognize DataFrame._AXIS_NAMES, so we re-write it here
```

Comment:
What error was this throwing? Surprised it wouldn't see this

Reply:
I think _AXIS_NAMES is defined dynamically in a classmethod.

Reply:
yes, it likely needs typing, i would rather do that and remove this from here (followup ok)